docs/database/0006_queue_content.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE queue_content_base (
+    id SERIAL PRIMARY KEY,
+    account_id INTEGER NOT NULL REFERENCES account(id) ON DELETE CASCADE,
+    position numeric_20_11 NOT NULL
```

Comment:
Again-for the numeric data type. Are you anticipating the numbers to be large for position of "queued content?" What would this table hold specifically? Queued uploads to rss feeds? Just trying to clarify things for myself  !

Reply:
Good question, and I got the idea for [this blog post](https://begriffs.com/posts/2018-03-20-user-defined-order.html). @suorcd and I have discussed this quite a bit, and aren't 100% set on if we want to go this way or not.

The reasoning is basically 1) using integer for position can lead to problems, as reordering items in the list means every item after that item also needs its position updated. 2) using a decimal column allows you to reorder by fitting a reordered item between 2 positions, by calculating the middle point between the two.

An alternative to this could be a jsonb column that maintains the position order data on a higher order table (like playlist), which may be doable for our use case, but we'll lose some referential benefits.

Reply:
In this particular case, I think you are good to use the numeric datatype. Looking at this again, it makes more sense from a positioning perspective.